CONTRIBUTOR-CHECKLIST.md

```diff
@@ -0,0 +1,100 @@
+## OpenWhisk Package On-boarding Checklist (In-Progress)
+
+_For process questions, please contact the owners:
+Matt Rutkowski ([mrutkows@us.ibm.com](mailto:mrutkows@us.ibm.com)),
+Carlos Santana ([csantana@us.ibm.com](mailto:csantana@us.ibm.com))_
+
+1. **Contributor License Agreement (CLA)**
+  1. **Contributors**
+    * **Sign the CLA**: All contributors MUST sign and submit a CLA. _This includes *all developers of the code submission** NOT just the submitter._
+      * This is either the [CLA-Individual.md](https://github.com/openwhisk/openwhisk/blob/master/CLA-INDIVIDUAL.md) or the [CLA-CORPORATE.md](https://github.com/openwhisk/openwhisk/blob/master/CLA-CORPORATE.md) as contained in the main OpenWhisk repository in GitHub.
+    * **Verify CLA received**: Verify the CLAs have been received and are complete by **_both_** **Stephen Fink** (sjfink@us.ibm.com) and **Rodric Rabbah** (rabbah@us.ibm.com) as identified in the CLA instructions.
+  2. **OpenWhisk Admins**
+     * OpenWhisk admins will not accept pull requests from contributors unless CLAs from all contributors for the submission are received.
+[comment]: <=========================================================================================>
+2. **Open Source Approval**
+  1. **Contributors**
+    * In general, all open source contributions are in accordance to the Contributor License Agreements (CLAs) and submitted under the [LICENSE.txt](https://github.com/openwhisk/openwhisk/blob/master/LICENSE.txt) file of the OpenWhisk project (i.e., Apache 2 license).
+    * **Email Contributor list**: Email all contributor names, company affiliation and their email addresses to **Richard Jacob** ([richard.jacob@de.ibm.com](mailto:richard.jacob@de.ibm.com)).
+      * _Copy:_ Andreas Nauerz ([andreas.nauerz@de.ibm.com](mailto:andreas.nauerz@de.ibm.com)), Matt Rutkowski ([mrutkows@us.ibm.com](mailto:mrutkows@us.ibm.com)), Carlos Santana ([csantana@us.ibm.com](mailto:csantana@us.ibm.com))
+  2. **OpenWhisk Admins**
+    * OpenWhisk admins will not accept pull requests on new packages unless all contributor names are received.
+[comment]: <=========================================================================================>
+3. **Code Scan**
+  1. **OpenWhisk Admins**
+    * OpenWhisk admins MUST need to run code scan tools (i.e., [CSAR](https://w3-connections.ibm.com/wikis/home?lang=en-us#!/wiki/W7c33090f777f_4d05_bda9_b9d8b79b0c7c/page/Download%20CSAR)) for any package submitted if the package references external packages, libraries or files. The tool will identify legal concerns for accepting the code which MUST be resolved prior to accepting the submission.  This includes "hits" for _Keywords_, _Licenses_, _File imports_ and other concerning _anomalous references_.
```

Comment:
why is an internal link documented here? most of these instructions are for ibmers, should this be in our enterprise git instead?